indra/newview/llpanelprofile.cpp

```diff
@@ -1173,20 +1173,22 @@ void LLPanelProfileSecondLife::fillAgeData(const LLAvatarData* avatar_data)
         userAgeCtrl->setValue(register_date);
     }
 
+    BOOL showHideAgeCombo = FALSE;
     if (getSelfProfile())
     {
-        F64 birth = avatar_data->born_on.secondsSinceEpoch();
-        F64 now = LLDate::now().secondsSinceEpoch();
-        if (now - birth > 365 * 24 * 60 * 60)
+        if (LLAvatarPropertiesProcessor::getInstance()->isHideAgeSupportedByServer())
```

Comment:
either isHideAgeSupportedByServer() shouldn't be static or LLAvatarPropertiesProcessor::isHideAgeSupportedByServer()